daisy/workflow/step_create_disks.go

```diff
@@ -32,7 +33,7 @@ type CreateDisk struct {
 	compute.Disk
 
 	// Size of this disk.
-	SizeGb string `json:",omitempty"`
+	SizeGb string `json:"sizeGb,omitempty"`
```

Comment:
The nested structs define sizeGb (and metadata below) is explicitly defined by the nested compute.Disk. Which means that SizeGb is unmarshalled properly into our CreateDisk, but sizeGb is unmarshalled incorrectly into the compute.Disk.

By explicitly tagging here, this unmarshals both sizeGb and SizeGb into CreateDisk instead of the compute.Disk.